wls-broadcast-service/src/test/java/de/muenchen/oss/wahllokalsystem/broadcastservice/integration/BroadcastIntegrationTest.java

```diff
@@ -0,0 +1,296 @@
+        mvc.perform(get(GETMESSAGE_URL + wahlbezirkID)
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isNotFound())
+                .andExpect(result -> {
+                    Map<String, Object> mmodel = result.getModelAndView().getModel();
+                    Assertions
+                            .assertThat(mmodel)
+                            .containsKey("wlsException")
+                            .extracting("wlsException")
+                            .isInstanceOf(TechnischeWlsException.class)
+                            .extracting("code", "serviceName")
+                            .contains("999", serviceOid);
+
+                    Assertions
+                            .assertThat(mmodel)
+                            .containsKey("wlsException")
+                            .extracting("wlsException.cause")
+                            .isInstanceOf(NoResourceFoundException.class);
+                });
+    }
+
+    @Test
+    public void getMessageNoContentIntegrationTest() {
+        log.debug("#GetMessageNoContentIntegrationTest");
+
+        ServletException thrownException = null;
+        try {
+            mvc.perform(
+                    get(GETMESSAGE_URL + "123")
+                            .contentType(MediaType.APPLICATION_JSON_UTF8)
+                            .accept(MediaType.APPLICATION_JSON));
+        } catch (Exception e) {
+            thrownException = (ServletException) e;
+        }
+
+        Assertions.assertThat(thrownException.getCause())
+                .isNotNull()
+                .isInstanceOf(FachlicheWlsException.class)
+                .hasMessageContaining("No message found")
+                .extracting("code", "serviceName")
+                .contains("204", serviceOid);
+    }
+
+    @Test
+    public void deleteIntegrationTest() throws Exception {
+        log.debug("#deleteIntegrationTest");
+
+        Message message = Testdaten.createMessage("123", "Das ist ein Test", LocalDateTime.now());
+        messageRepository.save(message);
+
+        List<Message> foundMessages = messageRepository.findByWahlbezirkID("123");
+        Message foundMessage = foundMessages.stream().findFirst().get();
+        Assertions.assertThat(foundMessage).isNotNull();
+
+        // @formatter:off
+        MockHttpServletResponse result =
+                mvc.perform(
+                        post(DELETE_URL + foundMessage.getOid())
+                                .contentType(MediaType.APPLICATION_JSON_UTF8).accept(MediaType.APPLICATION_JSON)).andReturn().getResponse();
+        // @formatter:on
+
+        int status = result.getStatus();
+        Assertions.assertThat(status).isEqualTo(200);
+        log.info("Result > Status: {} ", status);
+
+        foundMessages = messageRepository.findByWahlbezirkID("123");
+        Assertions.assertThat(foundMessages).size().isEqualTo(0);
+    }
+
+    @Test
+    public void deleteIntegrationTestBadFormatUUID() {
```

Comment:
Schreibt man mit Junit 5 nicht mehr

```suggestion
    void deleteIntegrationTestBadFormatUUID() {
```